### ISSUE TITLE
Support RDS format in R package

### ISSUE THREAD

**Issue by khrisanfov** [catboost/catboost#91]
```
catboost.fit <- train(....)
head(predict(catboost.fit, X_test)) # it works fine
saveRDS(catboost.fit, "catboost.fit.rds")
catboost.fit <- readRDS("catboost.fit.rds")
head(predict(catboost.fit, X_test)) # caught segfault
```

 ```
*** caught segfault ***
address 0x48, cause 'memory not mapped'

Traceback:
 1: .Call("CatBoostGetModelParams_R", model$handle)
 2: catboost.get_model_params(modelFit)
 3: method$predict(modelFit = modelFit, newdata = newdata, submodels = param)
 4: predictionFunction(method = object$modelInfo, modelFit = object$finalModel,     newdata = newdata, preProc = object$preProcess)
 5: predict.train(catboost.fit, X_test)
 6: predict(catboost.fit, X_test)
 7: head(predict(catboost.fit, X_test))
```

**Comment by a4tunado** [comment]
RDS format is not supported yet. Use catboost.save_model and catboost.load_model instead.

**Comment by annaveronika** [comment]
This is implemented in our release 0.10
